resource/ui/MainWindow.ui

```diff
@@ -117,5 +105,8 @@
         </child>
       </object>
     </child>
+    <child type="titlebar">
```

Comment:
Could you please move titlebar child to the top again? You can use glade to edit ui files easily btw :)

Reply:
Hey, I used Glade and it moved it automatically. I don't see an option in the GUI. Do you want me to move it manually?

Reply:
Oh, you are using the very recent version of Glade. Then, this is not very important, it should stay like this.